### PR TITLE
[docs] Fix formatting issues in multiple files

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Detected formating issues in multiple files. This PR fixes all of them.

# Test Plan

By running `yarn run prettier`.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/config-plugins/development-and-debugging.mdx [hunk 1]
```diff
@@ -637,16 +636,16 @@ This does come with a couple of caveats:
...
-      "expo-camera"
-    ]
-  }
-  ```
+{
+  "plugins": [
+    "expo-camera"
+  ]
+}
+```
```
Comment (Simek) [inline docs/pages/config-plugins/development-and-debugging.mdx:648]: This was indented on purpose, so the code block appears nested under `3.` list element.

Screenshot 2024-06-30 083050

Comment (amandeepmittal) [reply docs/pages/config-plugins/development-and-debugging.mdx:648]: If that's the case, I'll revert it and add prettier-ignore rule.

Comment (amandeepmittal) [reply docs/pages/config-plugins/development-and-debugging.mdx]: I've reverted this change and added the prettier-ignore comment for this code snippet block. FYI, this doesn't have an effect on user-facing docs (whether we change it or not):

CleanShot 2024-06-30 at 18 44 18@2x

Link to the section: https://docs.expo.dev/config-plugins/development-and-debugging/#expo-install

Comment (Simek) [reply docs/pages/config-plugins/development-and-debugging.mdx]: We are missing one more space per line.

Comment (amandeepmittal) [reply docs/pages/config-plugins/development-and-debugging.mdx]: I overlooked that, my bad. Let me fix that!

Comment (amandeepmittal) [reply docs/pages/config-plugins/development-and-debugging.mdx]: Fixed it (also thanks for the tip to use shell code language.

CleanShot 2024-06-30 at 23 53 37

**Review by Simek — APPROVED** [review]

**Pull request merged**